.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -70,3 +70,15 @@ This change added tests and can be verified as follows:
 
   - Does this pull request introduce a new feature? (yes / no)
   - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
+
+----
+
+# Review Progress <!-- NOTE: DO NOT REMOVE THIS SECTION! -->
+
+* [ ] 1. The contribution is well-described.
+* [ ] 2. There is consensus that the contribution should go into to Flink.
```

Comment:
Can you give some examples to show what makes a consensus, let developers have a good understanding?

Reply:
Hi @KurtYoung, the consensus aspect is described in the [Pull Request Review Guide](https://flink.apache.org/reviewing-prs.html#is-there-consensus-that-the-change-or-feature-should-go-into-flink) that is linked below. 

Do you think this is sufficient or does it need more clarification?